lib/pg_ha_migrations/safe_statements.rb

```diff
@@ -506,6 +522,14 @@ def _type_is_enum(type)
     ActiveRecord::Base.connection.select_values("SELECT typname FROM pg_type JOIN pg_enum ON pg_type.oid = pg_enum.enumtypid").include?(type.to_s)
   end
 
+  def _ensure_empty_table!(table)
```

Comment:
Do you think this would be useful as a public method?

Reply:
If so, we'd need to make the error message a bit more generic

Reply:
Addressed in https://github.com/braintree/pg_ha_migrations/pull/93/commits/058c4d94367fe3679deff0fc8ceacaaa0f79fccd